### PR TITLE
Create bitlaunch.io.xml

### PR DESCRIPTION
**Pull request by BitLaunchIO**
## Type of change
- [x] New Ruleset

### CODE REVIEW

**Comment by pipboy96** [comment]
@BitLaunchIO Thanks for contribution! Since you are the website owner, it's recommended that you use HSTS preloading instead. It's more effective since it doesn't require the user to have HTTPS Everywhere installed, as long as they are on a modern version of Chrome or Firefox.

File: src/chrome/content/rules/bitlaunch.io.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="BitLaunch.io">
+	<target host="bitlaunch.io" />
+	<target host="www.bitlaunch.io" />
+  <target host="app.bitlaunch.io" />
+	<target host="pay.bitlaunch.io" />
```
Comment (pipboy96) [inline src/chrome/content/rules/bitlaunch.io.xml:5]:
```suggestion
	<target host="bitlaunch.io" />
	<target host="www.bitlaunch.io" />
	<target host="affiliates.bitlaunch.io" />
	<target host="analytics.bitlaunch.io" />
	<target host="app.bitlaunch.io" />
	<target host="blacklist.bitlaunch.io" />
	<target host="developers.bitlaunch.io" />
	<target host="pay.bitlaunch.io" />
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

**Comment by BitLaunchIO** [comment]
@pipboy96 Thank you.

We are observing an SSL stripping MITM attack on our payment endpoint via malicious Tor exit relays, in which the cryptocurrency payment address is replaced with that of the attacker. HSTS is now enabled. We'd still greatly appreciate consideration for merging so that all eventualities are covered.

**Comment by pipboy96** [comment]
@BitLaunchIO Also, an unrelated question: which datacenter are BitLaunch VPSes located in?

**Review by pipboy96 — APPROVED** [review]
LGTM

**Pull request merged**